packages/admin/src/routes/GetAdminUsers.route.ts

```diff
@@ -0,0 +1,19 @@
+import { ConduitRoute, ConduitRouteReturnDefinition } from '@conduitplatform/commons';
+import { ConduitRouteActions } from '@conduitplatform/grpc-sdk';
+import { Admin } from '../models';
+
+export function getAdminUsersRoute() {
+  return new ConduitRoute(
+    {
+      path: '/admins',
```

Comment:
Should we also update create admin path as `POST @ /admin/admins` while at it (breaking change).